setup.py

```diff
@@ -1,28 +1,16 @@
-import os
 from setuptools import setup, find_packages
 
-VERSION_FILE = 'VERSION'
-
-
-def get_version():
-    """Returns the current gdc model version"""
-    current_dir = os.path.dirname(os.path.realpath(__file__))
-    version_path = os.path.join(current_dir, VERSION_FILE)
-
-    with open(version_path, 'r') as f:
-        return f.read().strip()
-
 
 setup(
-    name = 'gdcmodels',
-    version = get_version(),
-    packages = find_packages(exclude=[
+    name='gdcmodels',
+    version='1.0.0',
```

Comment:
Just a comment: I think this shows a broader issue on how we provide version numbers for projects. We might need to come up with a more flexible and automated way to increment and set this value for each release

Reply:
something like [bumpversion](https://github.com/peritus/bumpversion)?